### PR TITLE
This link seems like it's stale since 2021

### PR DESCRIPTION
**Pull request by richardanderson001**
## What does this PR do?
Remove resource, the link seems stale and not accepting applications since 2021.

### Why is this not valuable?
Stale, seems like very little use for DS.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
should it be updated to https://codalab.lisn.upsaclay.fr ?

**Comment by richardanderson001** [comment]
Good find, I'm guessing that's probably correct!

File: more/problem-sets-competitive-programming.md [hunk 1]
```diff
@@ -69,7 +69,6 @@
 ### Data science
 
 * [AIcrowd](https://www.aicrowd.com)
```
Comment (Thenlie) [inline more/problem-sets-competitive-programming.md:71]:
```suggestion
* [AIcrowd](https://www.aicrowd.com)
* [CrowdANALYTIX](https://codalab.lisn.upsaclay.fr/)
```
@richardanderson001 Feel free to update this PR to include the new link.

**Comment by richardanderson001** [comment]
thanks @Thenlie just updated

**Comment by Thenlie** [comment]
@richardanderson001 Thanks for updating this one! Just be sure to check the output of the free-programming-books-lint action. In this case it looks like there is a trailing slash on the URL which should be removed. 

image

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by richardanderson001** [comment]
thanks @eshellman!

**Pull request merged**